### PR TITLE
Make cursor a pointer over js-enhanced images

### LINKED ISSUE #808: Arrow buttons (hover/click)
At this point the arrow buttons seem to stay grey once clicked. Not sure what decisions around this is. 

Hover could be the grey, but once clicked, the button should revert back to white version.

screen shot 2017-04-24 at 10 58 59

### PR DESCRIPTION
**Pull request by davidpmccormick**
## What is this PR trying to achieve?
Showing users gallery images are clickable (scroll to the image) if JS is working.

Also blurring the slider controls after a click (but not a keypress) to remove the grey background of the focus state. Fixes #808.

## What does it look like?
screen shot 2017-04-26 at 17 28 59

### CODE REVIEW

**Comment by davidpmccormick** [comment]
Apologies for committing after the approval – please re-review.

File: client/js/components/content-slider.js [hunk 1]
```diff
@@ -365,11 +365,17 @@ const contentSlider = (el, options) => {
...
+    currentTarget.blur();
+  }
+
   setup();
 
   // Handle click
   sliderElements.prevControl.addEventListener('click', prevSlide, true);
+  sliderElements.prevControl.addEventListener('mouseup', blurCurrentTarget);
   sliderElements.nextControl.addEventListener('click', nextSlide, true);
+  sliderElements.nextControl.addEventListener('mouseup', blurCurrentTarget);
```
Comment (jamesgorrie) [inline client/js/components/content-slider.js]: Hmm. Can't say I see the need for both?

Comment (davidpmccormick) [inline client/js/components/content-slider.js]: The `click` event fires on pressing enter or space, but we don't want to blur the control buttons if the user is navigating with the keyboard. The `mouseup` event fires after `click`, and (obviously) not relative to any keyboard events. This isn't clear – I'll add a comment.

Comment (jamesgorrie) [inline client/js/components/content-slider.js]: Being a 💩 - didn't realise it was for either control.

**Pull request merged**